main.py

```diff
@@ -92,6 +104,10 @@ def main(args):
     """
     # Initialise the video stream
     input_feed = InputFeeder(input_feed=args.input)
+    print('width = ' + str(args.width))
+    print('height = ' + str(args.height))
+    input_feed.cap.set(cv2.CAP_PROP_FRAME_WIDTH, args.width)
+    input_feed.cap.set(cv2.CAP_PROP_FRAME_HEIGHT, args.height)
```

Comment:
Perhaps, use the method that forms part of the `InputFeeder` class.
https://github.com/B0N0AI/pyvino_utils/blob/master/pyvino_utils/input_handler/input_feeder.py#L114+L119
```suggestion
input_feed.resize_cam_input(args.height, args.width)
```